appinfo/info.xml

```diff
@@ -7,10 +7,11 @@ More information is available in the text editor documentation.
 	</description>
 	<licence>AGPL</licence>
 	<author>Tom Needham, Björn Schießle</author>
-	<version>2.5.0</version>
+	<version>2.5.1</version>
 	<shipped>true</shipped>
 	<default_enable/>
 	<dependencies>
 		<nextcloud min-version="13" max-version="13" />
 	</dependencies>
+	<namespace>Files_Texteditor</namespace>
```

Comment:
PSR-4 should not contain underscores
and the camelcase is now an option since we have the namespace field here.